### DISCUSSION TITLE
FluxStore Listing – Business/Multivendor App

### DISCUSSION THREAD

**Discussion by decwap** [inspireui/support#5530] [Working on Features]
The feature for owners/vendors to add listing is very very essential in this app but it's missing. Please get us an app to help vendors/owners add listings. We need it because the Flustore Multivendor app doesn't work with Listeo.

**Comment by minhcasi** [comment]
@decwap - please use the Adding Listing feature by using Webview token

**Reply by ahmedkaazab** [reply]
Webview option to add the listing doesn't look professional , and no any ability for owner to get notifications or accept or reject bookings , everything else in the app is very good

**Reply by byndrha** [reply]
Maybe InspireUI Team should also add the "Quantity Buttons" to Extra Services item

**Reply by ahmedkaazab** [reply]
Totally Agree as well as In-App Purchases for Ios & Android

**Comment by ahmedkaazab** [comment]
AnyUpdates Gentlemen ??

**Comment by ahmedazabo2021** [comment]
it is A must Option in The App , Any Updates when it will be released ?

**Comment by Yan-Aleman** [comment]
Push notifications to the owner (listeo), about new bookings or pre-bookings. Will solve problems so that the owner will begin to respond and respond in a timely manner.
